### ISSUE TITLE
Change colors for `variable.other.member`

### ISSUE THREAD

**Issue by NikitaRevenco** [catppuccin/helix#56]
Imo there's a bit too much blue. Making `variable.other.member` the same colors as function is confusing. Having a different color for regular functions vs variables makes it easier to differentiate them. Could https://github.com/catppuccin/helix/pull/50 be reverted? Just my suggestion. Feel free to close this

## Before

![image](https://github.com/user-attachments/assets/1c0917ac-a19a-49fd-8001-4f639533877e)

## After

![image](https://github.com/user-attachments/assets/09240b5a-d6e1-425f-ab32-0c70d70589da)

**Comment by sgoudham** [comment]
Struct members should be `text` iirc

**Comment by uncenter** [comment]
> Struct members should be `text` iirc, seems like it's a side effect of #50

Correct. Helix doesn't differentiate between these, it's all the same `variable.other.member`. It boils down to an issue with the queries.
